benches/README.md

```diff
@@ -0,0 +1,71 @@
+
+* **Rescue Prime Optimized:**
+As specified [here](https://eprint.iacr.org/2022/1577) and implemented in this crate.
+
+* **BLAKE3:**
+As specified [here](https://github.com/BLAKE3-team/BLAKE3-specs/blob/master/blake3.pdf) and implemented in this crate.
+
+* **SHA256:**
+As specified [here](https://csrc.nist.gov/csrc/media/publications/fips/180/4/final/documents/fips180-4-draft-aug2014.pdf) and implemented [here](https://github.com/novifinancial/winterfell/blob/46dce1adf0/crypto/src/hash/sha/mod.rs).
+
+* **Poseidon:**
+As specified [here](https://eprint.iacr.org/2019/458.pdf) and implemented (without vectorized instructions) [here](https://github.com/mir-protocol/plonky2/blob/main/plonky2/src/hash/poseidon_goldilocks.rs).
+
+## Comparison and Instructions
+
+### Comparison
+We benchmark the above hash functions using two scenarios. The first is a 2-to-1 $(a,b)\mapsto h(a,b)$ hashing where both $a$, $b$ and $h(a,b)$ are the digests corresponding to each of the hash functions.
+The second scenario is that of sequential hashing where we take a sequence of length $100$ field elements and hash these to produce a single digest. The digests are $4$ field elements (i.e. 256-bit) for Poseidon, Rescue Prime and RPO, and an array `[u8;32]` for SHA256 and BLAKE3.
+
+#### Scenario 1: 2-to-1 hashing `h(a,b)` 
+
+| Function | Rp64_256 | RPO_256 | BLAKE3 | SHA256 | Poseidon |
+| -------- | -------- | -------- | -------- | -------- | -------- |
+| Apple M1 | Text     | Text     | Text     | Text     | Text     |
+| AMD      | Text     | Text     | Text     | Text     | Text     |
+| x86 (CPU)| Text     | Text     | Text     | Text     | Text     |
+| ...      | Text     | Text     | Text     | Text     | Text     |
+
+#### Scenario 2: Sequential hashing of 100 elements `h([a_0,...,a_99])`
+
+| Function | Rp64_256 | RPO_256 | BLAKE3 | SHA256 | Poseidon |
+| -------- | -------- | -------- | -------- | -------- | -------- |
+| Apple M1 | Text     | Text     | Text     | Text     | Text     |
+| AMD      | Text     | Text     | Text     | Text     | Text     |
+| x86 (CPU)| Text     | Text     | Text     | Text     | Text     |
+| ...      | Text     | Text     | Text     | Text     | Text     |
+
+### Instructions
+To run these benchmarks, first install `rustup` using the following [instructions](https://rustup.rs/). Then, install the Rust toolchain by invoking:
+
+```
+rustup install stable
+```
+After that, to run the benchmarks for RPO and BLAKE3, clone the current repository:
+
+```
+git clone https://github.com/0xPolygonMiden/crypto.git
+```
+ and then run:
+
+ ```
+ cd crypto
+ cargo bench --bench hash
+ ```
+
+To run the benchmarks for Rescue Prime, Poseidon and SHA256, clone the following [repository](https://github.com/Dominik1999/winterfell.git) as above, then checkout the `hash-functions-benches` branch:
+
+```
+cd winterfell
```

Comment:
I also need to use `cd crypto`

Reply:
I don't, have you tried it without `cd crypto` ?

Reply:
I would just say: "... check out the `hash-functions-benches` branch, and from the root directory of the repo run the following:`

This way we avoid describing how to navigate to the directory.